README.md

```diff
@@ -41,6 +41,16 @@ the asset manifest. This defaults to `[ 'js', 'css' ]`.
 _Note: This class provides default `contentFor`, `postprocessTree`, and `postBuild` hooks so be sure that you call
 `_super` if you override one of those methods._
 
+### Options
+
+##### noManifest
+* Type: `boolean`
+* If `true`, a <meta> tag which would eventually hold the manifest will not be inserted into the DOM.
+
+##### noManifestLookup
```

Comment:
Does the flag actually not lookup the manifest? Or is it just going to ignore any errors arising from a lookup? What is the default value?

Reply:
If `noManifestLookup` is set to `true`, then there will be no manifest lookup. The default value will be `undefined`, meaning the manifest will be looked up by default (backwards compatible)